scss/_variables.scss

```diff
@@ -25,12 +25,15 @@ $navy: #164f85;
 
 $gray-lightest: #f1f1f1; // U.S. Web Design Standards color gray-lightest //
 $gray-light: #e8e8e8;
-$gray-medium: #d6d7d9; // U.S. Web Design Standards color gray-mediumer //
+$gray-medium: #d6d7d9; // U.S. Web Design Standards color gray-lighter //
 $gray: #aeb0b5; // U.S. Web Design Standards color gray-medium //
 $gray-dark: #5b616b; // U.S. Web Design Standards color gray //
 
 
 $disabled: #737373;
+$green-light: #4AA564; // U.S. Web Design Standards color green-light //
+$green-dark: #287538;
```

Comment:
@noahmanger if the red below is `$error`, should these greens be named something like `$success` or `$confirm` ?